### ISSUE TITLE
title of the webpage

### ISSUE THREAD

**Issue by wenbinyu-PU** [banghuazhao/swiftcomp-flutter#22]
If I want to change the title from SwiftComp to Composites AI, where should I make changes? index.html <title>Composites AI</title> or other places, if this file is automatically generated?

![image](https://github.com/user-attachments/assets/2c330284-101a-4c5e-a1c5-d910a098adfa)

**Comment by banghuazhao** [comment]
Right, the place is index.html <title>Composites AI</title>. In double-checked with chatGPT, it is automatically generated when we first config the web. The subsequence build will not be changed. So we can safely change it.

**Comment by banghuazhao** [comment]
BTW, you can follow those steps to link the branch and commit to a GitHub issue, which enables automatic closing upon merging, follow these steps:

## Step 1: Create a Branch with the Issue Number
Include the issue number in the branch name to make it easier to track. For example, if the issue number is #22, you can name the branch:
```bash
git checkout -b Fix-issue-22-update-web-title
```
## Step 2: Edit the Title in index.html
Make your changes to the index.html file as described earlier.

## Step 3: Commit the Changes with an Issue Reference
In your commit message, reference the issue number and include keywords for automatic closing. For example:
```bash
git add .
git commit -m "Fix #22: Update webpage title to Composites AI"
```
Using "Fix #22" (or similar keywords like "Closes #22" or "Resolves #22") will automatically close the issue once this branch is merged.

## Step 4: Push the Branch to the Remote Repository

## Step 5: Create a Pull Request

I will review the pull request and merge it. The pull request will also run github action to compile the project and run tests

**Comment by wenbinyu-PU** [comment]
can I resolve multiple issues using "Fix #22, #23: ...."?

**Comment by banghuazhao** [comment]
Yes, to close multiple issues in a single commit, you could use the closing keywords (e.g., "Fixes", "Closes") before each issue number. Here's the correct syntax:

* Correct: Fixes #22, fixes #23: [commit message]
* Correct: Closes #22 and closes #23: [commit message]
* Incorrect: Fix #22, #23: [commit message]

BTW, I have created the pull request for your branch: https://github.com/banghuazhao/swiftcomp-flutter/pull/23
